### DISCUSSION TITLE
Welcome to typescript-npm-package Discussions!

### DISCUSSION THREAD

**Discussion by jasonsturges** [jasonsturges/typescript-npm-package#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sendyputra** [comment]
Hello @jasonsturges I really appreciate this template, may you can added Automate versioning and package publishing with [release-it](https://github.com/release-it/release-it) it will help for Release Publishing. Thanks ☺️

**Reply by jasonsturges** [reply]
@sendyputra Cool, thanks for the note.  Sounds interesting, I'll check it out.
